cmd/beacon-chain/sync/backfill/flags/flags.go

```diff
@@ -0,0 +1,38 @@
+package flags
```

Comment:
We need to import the backfill flag to our feature package, since we want it as part of `--dev`. If you dont have a separate package, build fails due to dependency cycles